app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
     package="com.example.android.sampletvinput"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="25"/>
```

Comment:
Is there an warning that happens because of this? 21 needs to be the min version because that is the earliest version of the Android TV platform.

Reply:
The warning is given on newer gradle versions, on 3.1.0 there is no warning. The warning happens because in app.gradle also we have the min version ...
defaultConfig {
        applicationId "com.example.android.sampletvinput"
        minSdkVersion 21
        targetSdkVersion 27
        versionCode 3
        versionName "1.2"
    }

the same is duplicated in manifest as well.